iothub/service/src/Common/Exceptions/DeviceNotOnlineException.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.Azure.Devices.Common.Exceptions
+{
+    [Serializable]
```

Comment:
Why is this Serializable?

Reply:
this class is implemented as DeviceNotFoundException which is serializable